### DISCUSSION TITLE
How to update dependencies for a package that is both Deep package and js library at the same time?

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#24] [Q&A]
# Story
I have found out that in `@deep-foundation/capacitor-motion` library I need the latest `@deep-foundation/capacitor-device` library. I have installed it. I have realized that the version of `@deep-foundation/capacitor-device` dependency is changed only in package.json, but not in deep.json
# Question
What should we do? How should we sync dependencies versions of deep.json and package.json?

**Comment by FreePhoenix888** [comment]
@Konard

**Reply by konard** [reply]
@FreePhoenix888 we have `npm-pull`, `npm-release`, we might have `npm-install`. It should work as `npm i`, but also update dependency in `deep.json`.
